CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# geometry Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+### Added
+
+### Changed
+
+## 1.0.0 - 2017-04-05
```

Comment:
We have not pushed it yet. I could probably link to the revision and guess the correct URL. However, [Keep A Changelog](http://keepachangelog.com/en/0.3.0/) suggests linking the comparison between 2 consecutive versions.